Automattic/wp-calypso
# Testing new copy on the signup plans page

#### Proposed Changes

This PR includes copy changes to the the plans page shown in signup. The intent is to present each plan on its own merits, rather than shining such a negative light on Starter.

Here's a screenshot of the page:
![screencapture-calypso-localhost-3000-start-plans-2022-06-03-15_04_27](https://user-images.githubusercontent.com/35781181/171932502-4c9d2e7b-88da-41e0-8c74-b91209fb822c.png)

It is setup as an Explat test and restricted to EN + US so translations and local currency aren't an issue.

#### Testing Instructions
* Checkout this PR and start it locally.
* Assign yourself to the treatment in Explat experiment  `pricing_packaging_plans_page_copy_test`
* Navigate to signup in your local Calypso.
* You should see the updates on the plans step in Calypso.
* Try multiple viewports.
* Navigate to the plans page inside the the product to ensure it displays properly there as well.




*



Related to #
